src/Adapter/Product/ProductDataProvider.php

```diff
@@ -111,6 +112,17 @@ public function getQuantity($id_product, $id_product_attribute = null, $cache_is
         return Product::getQuantity($id_product, $id_product_attribute, $cache_is_pack);
     }
 
+    /**
+     * @param int $id_product
+     * @param int $id_product_attribute Optional
+     *
+     * @return string
+     */
+    public function getLocation($id_product, $id_product_attribute = null)
```

Comment:
I think you should use a default int value instead, as we already know 0 is the default value :+1: